lib/elastic_apm/instrumenter.rb

```diff
@@ -159,7 +159,7 @@ def start_span(
       return unless transaction.sampled?
       return unless transaction.inc_started_spans!
 
-      parent = parent || current_span || current_transaction
+      parent ||= current_span || current_transaction
```

Comment:
I would `parent ||= (current_span || current_transaction)` around the result, so it's not mistaken for `(parent ||= current_span) || current_transaction`